tests/scripts/thread-cert/Cert_8_1_01_Commissioning.py

```diff
@@ -65,7 +65,7 @@ def test(self):
 
         self.nodes[JOINER].interface_up()
         self.nodes[JOINER].joiner_start('openthread')
-        time.sleep(10)
+        time.sleep(15)
```

Comment:
As far as I know, this should have all logging enabled. Are there specific places in the code that I should add more logs?